AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml

```diff
@@ -37,5 +37,7 @@
         <element name="sortOption" type="button" selector="//div[@class='masonry-sorting'] //option[@value='{{sortOption}}']" parameterized="true"/>
         <element name="deleteMediaButton" type="button" selector="button#delete_files"/>
         <element name="licensedLabel" type="block" selector="//div[@class='masonry-image-column' and @data-repeat-index='0']//span[text()='Licensed']"/>
+        <element name="IncorrectSecretModalText" type="block" selector="//div[contains(text(), 'Login failed')]"/>
+        <element name="IncorrectSecretModalButton" type="button" selector="//span[text()='Ok']"/>
```

Comment:
I believe this way you validate if there is any 'Ok' (or 'Login failed') text on o page. I think it's better to have a specific selector and then validate text inside.

Reply:
I will fix that, thanks!

Reply:
@hws47a, I've fixed it. Please review when you can. Thank you!

Reply:
Hi @drpayyne, it looks much better.

Reply:
I personally think could be still better to use css selectors instead, use data-ui attribute instead of classes and/or reuse existing sections by cascading them.

Reply:
Also I think it could be better to reuse `AdobeStockSection.model` and check for content inside

Reply:
Don't think I can use `AdobeStockSection.modal` as this error is in a new independent modal with no keyword related to Adobe. This is the modal HTML content.

```
<aside role="dialog" class="modal-popup confirm                                 _show" aria-labelledby="modal-title-15" aria-describedby="modal-content-15" data-role="modal" data-type="popup" tabindex="0" style="z-index: 904; margin-left: 90px;">
    <div data-role="focusable-start" tabindex="0"></div>
    <div class="modal-inner-wrap" data-role="focusable-scope">
        <header class="modal-header">
            <h1 id="modal-title-15" class="modal-title" data-role="title">
                    Couldn't log you in
            </h1>
            <button class="action-close" data-role="closeBtn" type="button">
                <span>Close</span>
            </button>
        </header>
        <div id="modal-content-15" class="modal-content" data-role="content"><div>Login failed. Please check if <a href="https://adobe-stock.site/admin/admin/system_config/edit/section/system/">the Secret Key</a> is set correctly and try again.</div></div>
        <footer class="modal-footer">
            <button class="action-primary" type="button" data-role="action"><span>Ok</span></button>
        </footer>
    </div>
    <div data-role="focusable-end" tabindex="0"></div>
</aside>
```

Reply:
Not sure if I can use CSS selectors since the classes and attributes are too general and randomly generated. Only unique and constant aspect of this I could find is the error message and button text. Hence my solution to go with XPath.